src/main/scala/dameo/format/FEN.scala

```diff
@@ -1,31 +1,62 @@
 package strategygames.dameo.format
 
 import strategygames.Player
-import strategygames.dameo.PieceMap
-
-//TODO Dameo, work out the FEN structure for a Dameo game.
-//What do other platforms use?
-//Is there a standard for Dameo?
-//Are we able to use something similar to Draughts FENs (look in there)
-//Or do we want to invent our own, and have it more similar to chess?
-//Either way any FEN parsing wants to be done here. Look at other game logic FEN files
+import strategygames.dameo.{ Piece, PieceMap, Pos, Role }
+/*
+
+Dameo FEN:
+[colour to move]:W[white piece coords]:B[black piece coords]:Halfmoveclock:Fullmoves
+
+coords for men are a1,b1,c1
+other roles (kings, ghostman, ghostking, activeman, activeking) are prepended to the coord using
+a capital letter, e.g. Ka1,Gb1,Pb2. Ghosts are part of the FEN since we want to treat (partial)
+moves with ghosts in them as proper moves w.r.t. the frontend.
+
+ */
+
 final case class FEN(value: String) extends AnyVal {
 
   override def toString = value
 
+  def splitted: Array[String] = value.split(':')
+
   def player: Option[Player] =
-    value.split(':').lift(0) flatMap (_.headOption) flatMap Player.apply
+    splitted.lift(0) flatMap (_.headOption) flatMap Player.apply
 
   def initial = value == Forsyth.initial.value
 
-  // TODO Dameo set
-  def pieces: PieceMap = Map.empty
+  def player1: Player              = Player.apply(splitted(1).head).get
```

Comment:
Lots of unsafe at runtime functions here.
- Indexing splitted
- .head
- .get
- .tail
You can see the safe way to write functions like this in the function of `def player` on L24. (Use lift, headOption, and flatMaps etc)